build-scripts/profile_tool.py

```diff
@@ -44,6 +44,9 @@ def parse_args():
     parser_stats.add_argument("--implemented-ovals", default=False,
                         action="store_true", dest="implemented_ovals",
                         help="Show IDs of implemented OVAL checks.")
+    parser_stats.add_argument("--implemented-sces", default=False,
+                        action="store_true", dest="implemented_sces",
```

Comment:
Note that @pep8speaks complains about these lines, but they're matching the rest of the method so I'm inclined to leave them as-is and if we want to fix the entire method in a separate PR we can.